sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -109,6 +109,8 @@ class HiveSparkSubmitSuite
   }
 
   test("SPARK-8368: includes jars passed in through --jars") {
+    val warehousePath = System.getProperty("user.dir") + "/../../spark-warehouse"
```

Comment:
@srowen Thanks a lot for your help!
Yes, it is better to clean up the temporary path at the source. I will have a check and update the PR.
Thanks!